### ISSUE TITLE
Morph inline 

### ISSUE THREAD

**Issue by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#54]
I created a morph inline similar to the like example we discussed Wednesday. 

When 'like' is pressed the inflected forms show up, and when one is selected the window closes. 

I struggled to find a system of tags to make this happen for all verbs without making individual actions rules for each morph. Wondering if this is a place where multiple tags might help, but could not get it to work. 

[inline.csv](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/95b8bfbd07304656186cea76749e7c2f4bf584b2/morph_inline/Sofia_60_morph_inline.csv)

[inline.osdpi](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/95b8bfbd07304656186cea76749e7c2f4bf584b2/morph_inline/Sofia_60_morph_inline.osdpi)

**Comment by gbishop** [comment]
Here is my attempt at making what I think you wanted. It is trickier than I'd like and I think it might be easier to understand with more than the one tag. But this shows how you might do it with only one.
I also deleted some of the unused modal stuff.

https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/main/morph_inline/Sofia_60_morph_inline_gb.osdpi

https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/main/morph_inline/Sofia_60_morph_inline_gb.csv

**Comment by sofia-bg** [comment]
Maybe we can find a time to meet sometime for you to show me how to set it up with tags. I tried adding a tag_1 and doing something similar to what you did with the morph column. Of course I was thinking too much with language and hadn't considered using numbers which makes a lot more sense. 

When I open this .osdpi file there is not anything happening with those verbs. Tried re-loading the csv but that did not change anything. Seeing the csv and the actions helps though.
